name,value,relevant log,explanation
io.bytes.per.checksum,-1687820511,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null
	",value-hit
dfs.qjournal.http.read.timeout.ms,60000,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null
	",property-item-hit	read
ha.health-monitor.rpc.connect.max.retries,1,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null
	",value-hit
mapreduce.job.end-notification.retry.attempts,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null
	",value-hit
yarn.client.failover-retries,0,"ERROR	Thread Thread[eventHandlingThread,5,main] threw an Exception.org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null
	",value-hit
io.bytes.per.checksum,-1687820511,"ERROR	Task: attempt_1701312551309_0287_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",value-hit
mapreduce.task.stuck.timeout-ms,600000,"ERROR	Task: attempt_1701312551309_0287_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",property-item-hit	task
ha.health-monitor.rpc.connect.max.retries,1,"ERROR	Task: attempt_1701312551309_0287_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",value-hit
mapreduce.job.end-notification.retry.attempts,0,"ERROR	Task: attempt_1701312551309_0287_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",value-hit
yarn.client.failover-retries,0,"ERROR	Task: attempt_1701312551309_0287_r_000000_0 - exited : org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",value-hit
io.bytes.per.checksum,-1687820511,"INFO	Diagnostics report from attempt_1701312551309_0287_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",value-hit
ha.health-monitor.rpc.connect.max.retries,1,"INFO	Diagnostics report from attempt_1701312551309_0287_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",value-hit
mapreduce.job.end-notification.retry.attempts,0,"INFO	Diagnostics report from attempt_1701312551309_0287_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",value-hit
yarn.client.failover-retries,0,"INFO	Diagnostics report from attempt_1701312551309_0287_r_000000_0: Error: org.apache.hadoop.HadoopIllegalArgumentException: Invalid checksum type: userOpt=null, default=CRC32C:-1687820511, effective=null	",value-hit
ha.health-monitor.rpc.connect.max.retries,1,INFO	1 failures on node ed761ad59d44,value-hit
io.bytes.per.checksum,-1687820511,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	io
ha.health-monitor.rpc.connect.max.retries,1,"INFO	Service JobHistoryEventHandler failed in state STOPPEDjava.lang.NullPointerException
	",property-item-hit	ha
io.bytes.per.checksum,-1687820511,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	io
ha.health-monitor.rpc.connect.max.retries,1,"WARN	When stopping the service JobHistoryEventHandlerjava.lang.NullPointerException
	",property-item-hit	ha
io.bytes.per.checksum,-1687820511,"WARN	Graceful stop failed. Exiting..java.lang.NullPointerException
	",property-item-hit	io
io.bytes.per.checksum,-1687820511,INFO	Exiting with status 1: java.lang.NullPointerException,property-item-hit	io
ha.health-monitor.rpc.connect.max.retries,1,INFO	Exiting with status 1: java.lang.NullPointerException,value-hit
